include/m_name_table.h

```diff
@@ -149,4 +149,14 @@
 #define DEAD_GOLD_SAPLING (GOLD_TREE + 1)
 /* end of environmental objects */
 
+#define SNOWMAN0 40968
```

Comment:
This probably should be 0xZZZZ format since it's u16

Reply:
This value in hex is `0xA008`, which isn't very meaningful either, unless I'm missing something.
I think just sticking with one would be fine

Also, all the `SNOWMANX` could be an enum, what do you think?

Reply:
I think that 0xAZZZ correspond to some category of actors, since I've seen this trend with structures and npcs
While the list is incomplete, I think having a define at the moment could be good, and switch to an enum when we have all the IDs figured out

Reply:
Ohhhh, so this is part of whatever is going on with actors, I see.
In that case yeah, I agree using hex and defines is better, at least for now